ways-of-working/index.md

```diff
@@ -0,0 +1,89 @@
+# Ways Of Working (WOW)
+
+**Status**: first sketch, work in progress, request for collaboration
+
+**Date**: 2025-03-21
+
+**Governance**: To Be Discovered; potentially a combo of this repo partipants, DHCW CISO, NHS Wales UCB peers, etc.
+
+## Situation - Context and Problem Statement
+
+* We want to encourage high-trust high-speed participation in our org and our repos, and help shape what good participation cna look like, rather than just a blank slate.
+
+* Broadly this is known as our "ways of working", and some people and companies may also know this as a mix of principles, values, whys, tenets, ground rules, aspirations, norms, working agreements, culture, etc.
+
+## Background - Decision Drivers
+
+* We want to get good-enough quality work in place now, so we can try it out, ask for feedback, get help with it with Capacitas consultants who have two more weeks with us, etc.
+
+* We want to align with industry standards (not reinvent the wheel).
+
+## Assessment - Considered Options
+
+For immediate ship, we want to align with what we know works well broadly (which Joel has) and what works well for NHS (which NHS England has):
+
+* [Ways of Working](http://github.com/joelparkerhenderson/ways-of-working) – Joel's many-year multi-client aggregation.
+
+* [NHS England GitHub ways of working](https://github.com/nhsengland/github-WoW)
+
+To expand this ADR during the work-in-progress, here are a bunch of additional options, and we could/should add more as we discover them:
+
+* [Team Norms, Working Agreements, and Simple Rules by Esther Derby](doc/team-norms-by-esther-derby/)
```

Comment:
This link and the ones below are referring to a 'doc/' folder, believe these are intended to link to https://github.com/joelparkerhenderson/ways-of-working/tree/main/doc or did you intend to copy these into this repo; either way needs fixing.

Reply:
Fixed into links, thanks. Do you have a preference of link vs. copy? (I don't have a preference)